bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts

```diff
@@ -16,9 +16,9 @@ import {ScrollService} from '../../services/scroll.service';
 })
 export class ContactComponent implements OnInit {
   public errorMessage: string;
-  public errorMessages: [string?] = [];
+  public errorMessages: [string?] = ['Phone number is invalid.'];
```

Comment:
Don't want to hard code error message here.  Only want to add error message when we know for sure what the error message is and only added error message to array at the time of error.